solr/webapp/web/js/angular/services.js

```diff
@@ -58,7 +58,7 @@ solrAdminServices.factory('System',
   }])
 .factory('Logging',
   ['$resource', function($resource) {
-    return $resource('admin/info/logging', {'wt':'json', '_':Date.now()}, {
+    return $resource('admin/info/logging', {'wt':'json', 'nodes': 'all', '_':Date.now()}, {
```

Comment:
I'm very unfamiliar with the admin UI code structure.  Can you explain why nodes=all needed to be set in two places?

Reply:
@NazerkeBS ?

Reply:
@dsmiley, I set nodes=all to only one place which is in services.js file

Reply:
Okay; I see you improved that from an earlier commit.

@janhoy maybe you could take a look at this please. Should nodes=all should only be added when setLevel is submitted, not for all logging handler interactions.  Do you think this is done correctly here?

Reply:
I think nodes=all should be added only to the `setLevel` call. Should be easy to check in browser debugger that it sends correct request.

Reply:
I verified this is a problem now!  Thanks for the tip Jan; I forgot how easy it is to verify :-)

Reply:
Thank Jan and David! Just updated the PR.